Update release history and binaries for Release 1.10.0 (#71)
instana/nginx-tracing: README.md

  The output above shows that the module version 1.21.3 is required for NGINX Plus R25 P1.

> - The OpenSSL version, as shown by the openssl version command:

Reviewer: ```suggestion
- The OpenSSL version, as shown by the `openssl version` command:
```

Author: Updated.

  ```sh
  # openssl version
  OpenSSL 3.0.2 15 Mar 2022 (Library: OpenSSL 3.0.2 15 Mar 2022)
  ...
  ```

>   The output above shows that the module version 3.0.2 is required for libinstana_sensor_ssl3x.so.

Reviewer: ```suggestion
  The output above shows that the module `libinstana_sensor_ssl3x.so` is required for OpenSSL version 3.0.2.
```

Author: Updated.

  ```sh
  # openssl version
  OpenSSL 1.1.1f  31 Mar 2020
  ...
  ```

>   The output above shows that the module version 1.1.1f is required for libinstana_sensor_ssl1.1x.so.

Reviewer: ```suggestion
  The output above shows that the module `libinstana_sensor_ssl1.1x.so` is required for OpenSSL version 1.1.1f.
```

Author: Updated.

- The Libc variant used in your distribution (`glibc` or `musl`); you likely use `glibc`, unless you are using Alpine as base-image for your containers, in which case, it's `musl`.
- (In some cases) the particular distribution (when the build used in some official packages is different enough to require bespoke adjustments on Instana side)

> - The OpenSSL variant of libinstana_sensor.so (either libinstana_sensor_ssl1.1x.so or libinstana_sensor_ssl3x.so) is to support serverless monitoring with OpenSSL 1.1.x and OpenSSL 3.x.x. Choose correct OpenSSL variant for serverless monitoring; otherwise use libinstana_sensor.so.

Reviewer: ```suggestion
- The OpenSSL variant of `libinstana_sensor.so` (either `libinstana_sensor_ssl1.1x.so` or `libinstana_sensor_ssl3x.so`) is to support serverless tracing with OpenSSL 1.1.x or OpenSSL 3.x.x. Choose the correct OpenSSL variant for serverless tracing. Otherwise, use `libinstana_sensor.so`.
```

Author: Updated.

The list of binaries and download links is available on the [Binaries](binaries.md) page.

### Copy the Binaries

> From the  libinstana_sensor.so and OpenSSL variants of libinstana_sensor.so, choose the correct OpenSSL variant for serverless monitoring; if none is needed, use the standard libinstana_sensor.so.

Reviewer: ```suggestion
From the `libinstana_sensor.so` and OpenSSL variants of `libinstana_sensor.so`, choose the correct OpenSSL variant for serverless tracing. If none is needed, use the standard `libinstana_sensor.so`.
```

Author: Updated.

> Ensure that both the chosen libinstana_sensor.so and ngx_http_opentracing_module.so are placed on a filesystem that the NGINX process can access, both in terms of locations as well as file permissions.

Reviewer: ```suggestion
Ensure that both files, the chosen `libinstana_sensor.so` and `ngx_http_opentracing_module.so`, are placed on a filesystem that the NGINX process can access, both in terms of locations as well as file permissions.
```

Author: Updated.

## Release History

### 1.10.0 (2024-10-14)

>   * New Feature: Integrate serverless monitoring capabilities for NGINX.

Reviewer: ```suggestion
  * New Feature: Integrate serverless tracing capabilities for NGINX.
```

Author: Updated.
